convert2rhel/systeminfo.py

```diff
@@ -354,15 +375,14 @@ def _get_releasever(self):
         releasever_cfg = self._get_cfg_opt("releasever")
         try:
             # return config value or corresponding releasever from the RELEASE_VER_MAPPING
-            return releasever_cfg or RELEASE_VER_MAPPING[".".join(map(str, self.version))]
+            return releasever_cfg or RELEASE_VER_MAPPING[repr(self.version)]
```

Comment:
Same here, should be possible to do this (though not super sure)
```suggestion
            return releasever_cfg or RELEASE_VER_MAPPING[self.version]
```

Reply:
I was wrong
```
>>> class RelVer():
...     def __repr__(self):
...             return "8.4"
...
>>> ver = RelVer()
>>> ver
8.4
>>> RELEASE_VER_MAPPING[ver]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 8.4
>>> RELEASE_VER_MAPPING[repr(ver)]
'8.4'
```